methodology/README.md

```diff
@@ -1 +1,56 @@
-# TBD
+# Penetration Testing Methodologies
+
+Ensuring a comprehensive approach to penetration testing is essential for effective cybersecurity. The following guidelines outline our strategy and methodologies for achieving consistent and reliable results, adhering to established standards in the field.
+
+## Overview
+
+Penetration testing involves a systematic and organized strategy to evaluate the security posture of networks and systems. This approach mitigates the risk of haphazard results and provides a structured framework for addressing potential vulnerabilities.
+
+## Methodologies and Standards
+
+Understanding and implementing major documented methodologies and standards are crucial elements of our approach. This empowers us to formulate strategies that leverage established practices, enhancing accountability and defensibility in our results.
+
+### Testing Environments
+
+Penetration testing methodologies often categorize tests based on the level of information provided to the tester. Common testing environments include:
+
+**1. Unknown-Environment Test:**
+   - Limited information provided (e.g., domain names and IP addresses).
+   - Mimics an external attacker's perspective, starting with minimal knowledge.
+   - Enhances realism by withholding information from network support personnel.
+
+**2. Known-Environment Test:**
+   - Tester possesses significant information about the organization and its infrastructure.
+   - Includes network diagrams, IP addresses, configurations, user credentials, and source code if applicable.
+   - Aims to identify security vulnerabilities within a broader scope.
+
+**3. Partially Known Environment Test:**
+   - A hybrid approach between unknown- and known-environment tests.
+   - Testers may be provided credentials but not full documentation of the network infrastructure.
+   - Allows testing from an external attacker's perspective while retaining some internal insights.
+
+### Common Penetration Testing Methodologies
+
+Several widely recognized methodologies and standards guide our penetration testing efforts:
+
+**1. [MITRE ATT&CK Framework](https://attack.mitre.org/):**
```

Comment:
[MITRE ATT&CK](https://attack.mitre.org/) is not a pen testing methodology, but instead a collection of adversary tactics and techniques. However, we have it listed as the first methodology here. Perhaps add it as an additional resource at the end.

Reply:
addressed in https://github.com/The-Art-of-Hacking/h4cker/pull/187/commits/0a222faa90df6051d9b3528dcd3452c7916d4be5